sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/AtomicInsertTest.java

```diff
@@ -56,7 +56,7 @@ private static boolean shouldAttempt(int i, long count) {
   /** With sufficient block size, ClickHouse will atomically insert all or nothing. */
   @Test
   public void testAtomicInsert() throws SQLException {
-    int size = 1000000;
+    int size = 100000;
```

Comment:
why did you shrink the size?

Reply:
Fixing

Reply:
setting 1M cause insert error. it also works with 500K rows, but i have changed it to 100K a deeper investigation need to be done here.